compiler/rustc_codegen_llvm/src/gotoc/statement.rs

```diff
@@ -140,20 +144,23 @@ impl<'tcx> GotocCtx<'tcx> {
             ty::Bool => {
                 let jmp: usize = values[0].try_into().unwrap();
                 let jmp2 = if jmp == 0 { 1 } else { 0 };
-                Stmt::block(vec![
-                    v.cast_to(Type::bool()).if_then_else(
+                Stmt::block(
+                    vec![
+                        v.cast_to(Type::bool()).if_then_else(
+                            Stmt::goto(
+                                self.current_fn().labels()[targets[jmp2].index()].clone(),
+                                Location::none(),
+                            ),
+                            None,
+                            Location::none(),
+                        ),
                         Stmt::goto(
-                            self.current_fn().labels()[targets[jmp2].index()].clone(),
+                            self.current_fn().labels()[targets[jmp].index()].clone(),
```

Comment:
I was going to ask why this changed from `jmp2` to `jmp` but I think its an artifact of the way github splits the lines.